receivers/email/email.go

```diff
@@ -108,3 +112,21 @@ func (en *Notifier) Notify(ctx context.Context, alerts ...*types.Alert) (bool, e
 func (en *Notifier) SendResolved() bool {
 	return !en.GetDisableResolveMessage()
 }
+
+func readFile(path string) ([]byte, error) {
+	_, err := os.Stat(path)
```

Comment:
I think this check here is redundant because Open below will catch it too. Perhaps remove it?

Reply:
Ah, true. Didn't put too much thought into this function when moving it out of `Notify` as it's immediately going away in the next PR. I'll remove the `os.Stat`